packages/backend/src/studio.ts

```diff
@@ -66,6 +68,16 @@ export class Studio {
 
   public async activate(): Promise<void> {
     console.log('starting AI Lab extension');
+    this.telemetry = env.createTelemetryLogger();
+
+    // Ensure version is above minimum podman desktop version supported
+    if (!version || !satisfies(version, engines['podman-desktop'])) {
+      this.telemetry.logError('start.incompatible', {
+        version: version,
+        message: 'error activating extension on version below 1.8.0',
```

Comment:
you need to change the error message and replace 1.8.0 by the value of `engines['podman-desktop']`

Reply:
To avoid making the tests depend on the package.json values, which are going to change, I mocked the imports.